gnat2goto/driver/tree_walk.adb

```diff
@@ -1433,9 +1433,13 @@ package body Tree_Walk is
       -- Do_Pragma_Assert --
       ----------------------
 
-      procedure Do_Pragma_Assert (N : Node_Id; Block : Irep);
-      procedure Do_Pragma_Assert (N : Node_Id; Block : Irep) is
-         Assert_Irep : constant Irep := New_Irep (I_Code_Assert);
+      procedure Do_Pragma_Assert_or_Assume
+        (N : Node_Id; Block : Irep; Which : String);
```

Comment:
No problem. Thanks for the review, it is helpful. I don't mind the split into multiple parts.